src/sentry/static/sentry/app/views/projectInstall/createProject.tsx

```diff
@@ -358,5 +358,5 @@ const TeamSelectInput = styled('div')`
 
 const HelpText = styled('p')`
   color: ${p => p.theme.gray400};
-  max-width: 700px;
+  max-width: 760px;
```

Comment:
Magic number here is a little odd, but I'm getting it's just visual preference.

Reply:
Ya, just increased it so it didn't split across multiple lines